### DISCUSSION TITLE
2.8.7图片上传失败

### DISCUSSION THREAD

**Discussion by JeffreyZeng-dev** [ArtalkJS/Artalk#963] [Q&A]
更新之前使用2.8.3上传图片正常，更新以后出现图片上传错误！
"level":"ERROR","ts":"2024-06-11T18:18:47.130+0800","caller":"[handler/upload.go:178]","msg":"通过 upgit 上传图片失败","id":"6f7d1ff8-d14b-4bf8-9499-df8cf4e7a01a","path":"/api/v2/upload","method":"POST","ip":"240e:47c:660:deb8:c184:a1f7:8973:5dba","remote_addr":"172.22.0.1:39364","host":"artalk.uptoz.cn","referer":"","user_agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36 Edg/125.0.0.0","status":500}
回退到2.8.3版本后仍能正常上传图片

**Comment by qwqcode** [comment]
你好，从 v2.8.4 版本开始，为了增强安全性，Artalk 不再允许指定 UpGit 的可执行文件路径，请将其加入系统的环境变量中。😃

参考：https://artalk.js.org/guide/backend/img-upload.html#%E4%BD%BF%E7%94%A8-upgit-%E4%B8%8A%E4%BC%A0%E5%88%B0%E5%9B%BE%E5%BA%8A
